test/unit/config.spec.js

```diff
@@ -300,6 +300,16 @@ describe('config', () => {
       expect(config.preprocessors).not.to.have.property(resolveWinPath('*.coffee'))
       expect(config.preprocessors).to.have.property(resolveWinPath('/**/*.html'))
     })
+
+    it('should validate that the browser option is an array', () => {
+      var invalid = function() {
+        normalizeConfigWithDefaults({
+          browsers: ''
+        })
+      }
+
+      expect(invalid).to.throw(/Invalid configuration/)
```

Comment:
Try checking the full error message, to be sure it's the right error that is thrown:

```js
expect(invalid).to.throw('Invalid configuration: browsers option must be an array')
```